src/js/allowmic.js

```diff
@@ -2,6 +2,21 @@ var micAccess = document.getElementById("micAccess");
 var backgroundChange = document.getElementById("backgroundChange");
 var backUrl = document.getElementById("backUrl");
 var theValue;
+var warning = document.getElementById("warning");
+var customBackground = document.getElementById("customBackground");
+var checkLogin = localStorage.getItem("checkLogin");
+
+window.onload = () => {
+    console.log("The login status is " + checkLogin);
+    if(checkLogin === "true") {
```

Comment:
Also, check if `accessToken` is present. Currently, the `accessToken` is not being saved. I think it is misspelled, when I last checked while working on #165